scripts/docs/build-features-overview.js

```diff
@@ -0,0 +1,456 @@
+
+	saveGeneratedOutput( generatedOutput );
+
+	return parsedFiles.length;
+}
+
+/**
+ * Reads and parses all package metadata files, that match the `glob` pattern. The returned array is sorted alphabetically by package name.
+ *
+ * @returns {Array.<ParsedFile>}
+ */
+function parseFiles() {
+	return glob.sync( PACKAGE_METADATA_PATH_PATTERN )
+		.map( readFile )
+		.map( file => {
+			try {
+				return parseFile( file );
+			} catch ( error ) {
+				error.message = `Failed to parse ${ chalk.bold( file.path ) }\n${ error.message }`;
+
+				throw error;
+			}
+		} )
+		.sort( ( parsedFileA, parsedFileB ) => parsedFileA.packageName.localeCompare( parsedFileB.packageName ) );
+}
+
+/**
+ * Reads the package metadata file.
+ *
+ * @param {String} path An absolute file path.
+ * @returns {File}
+ */
+function readFile( path ) {
+	return {
+		path,
+		content: fs.readFileSync( path, 'utf-8' )
+	};
+}
+
+/**
+ * Parses the package metadata file.
+ *
+ * @param {File} file Contains file path and its content to parse.
+ * @returns {ParsedFile}
+ */
+function parseFile( file ) {
+	const metadata = JSON.parse( file.content );
+
+	const packageName = path.basename( path.dirname( file.path ) );
+
+	const plugins = preparePlugins( packageName, metadata.plugins );
+
+	return {
+		packageName,
+		plugins
+	};
+}
+
+/**
+ * Parses all plugins from package metadata file.
+ *
+ * @param {String} packageName Package name.
+ * @param {Array.<Plugin>} plugins Plugins to parse.
+ * @returns {Array.<ParsedPlugin>}
+ */
+function preparePlugins( packageName, plugins = [] ) {
+	return plugins
+		.map( plugin => {
+			const pluginName = plugin.docs ?
+				prepareFeatureLink( plugin ) :
+				plugin.name;
+
+			const pluginClassName = prepareApiLink( packageName, plugin );
+
+			const htmlOutput = plugin.htmlOutput ?
+				prepareHtmlOutput( plugin.htmlOutput ) :
+				[ '<p>None.</p>' ];
+
+			return {
+				name: `<p>${ pluginName }</p><p>${ pluginClassName }</p>`,
+				htmlOutput
+			};
+		} );
+}
+
+/**
+ * Creates link to the plugin's feature documentation.
+ *
+ * @param {Plugin} plugin Plugin definition.
+ * @returns {String}
+ */
+function prepareFeatureLink( plugin ) {
+	const link = /http(s)?:/.test( plugin.docs ) ?
+		plugin.docs :
+		`../../../${ plugin.docs }`;
+
+	return `<a href="${ link }" data-skip-validation>${ plugin.name }</a>`;
+}
+
+/**
+ * Creates link to the plugin's API documentation.
+ *
+ * @param {String} packageName Package name.
+ * @param {Plugin} plugin Plugin definition.
+ * @returns {String}
+ */
+function prepareApiLink( packageName, plugin ) {
+	const shortPackageName = packageName.replace( /^ckeditor5-/g, '' );
+
+	const packagePath = plugin.path
+		.replace( /(^src\/)|(\.js$)/g, '' )
+		.replace( /\//g, '_' );
+
+	const link = `../../../api/module_${ shortPackageName }_${ packagePath }-${ plugin.className }.html`;
+
+	return `<a href="${ link }" data-skip-validation><code class="nowrap">${ plugin.className }</code></a>`;
```

Comment:
Ideally, we could skip validation only for CF and internal packages.

Reply:
Done. The `data-skip-validation` attribute is now added only to links from `external` folder.